tools/preparetranslations.php

```
@@ -31,15 +31,15 @@
         $translations[$text['name']] = true;
     }
-    if ($text['description'] && array_key_exists($text['description'], $translations) === false) {
+    if ($text['description'] && !isset($translations[$text['description']])) {
         $translateFileContents .= "_('" . str_replace("'", "\'", $text['description']) . "');\n";
         $translations[$text['description']] = true;
     }
 }
 
 file_put_contents('../src/translations.php', $translateFileContents);
-$apper->addStatusMessage(sprintf(_('%d language strings exported'), count($translations)), 'success');
+$apper->addStatusMessage(sprintf(count($translations) === 1 ? LANG_STRING_EXPORTED_SINGULAR : LANG_STRING_EXPORTED_PLURAL, count($translations)), 'success');
```

PHPStan (2.2.5)

[error] Constant LANG_STRING_EXPORTED_PLURAL not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)